OpenSim/Common/TimeSeriesTable.h

```diff
@@ -250,15 +250,15 @@ class TimeSeriesTable_ : public DataTable_<double, ETY> {
 
     \param time Value to search for.
     \param restrictToTimeRange  When true -- Exception is thrown if the given
-                                value is out-of-range of the time column.
+                                value is out-of-range of the time column. A value
+                                within SimTK::SignifcantReal of a time column
+                                bound is considered to be equal to the bound.
```

Comment:
@tkuchida and @chrisdembia, updated to include the fact that some slop is used to evaluate equality with the time bounds.